php-frontend/src/test/java/org/sonar/php/tree/impl/statement/EnumCaseTreeTest.java

```diff
@@ -54,4 +55,17 @@ public void enum_case_with_value() {
     assertThat(tree.equalToken()).hasToString("=");
     assertThat(tree.value().is(Tree.Kind.REGULAR_STRING_LITERAL)).isTrue();
   }
+
+  @Test
+  public void enum_case_keywords_as_identifiers() {
+    EnumCaseTreeImpl tree = parse("case NEW = 1;", PHPLexicalGrammar.ENUM_CASE);
+    assertThat(tree.equalToken()).hasToString("=");
+    assertThat(tree.value().is(Tree.Kind.NUMERIC_LITERAL)).isTrue();
+    assertThat(((LiteralTree) tree.value()).value()).isEqualTo("1");
+
+    tree = parse("case CLASS = 'a';", PHPLexicalGrammar.ENUM_CASE);
+    assertThat(tree.equalToken()).hasToString("=");
+    assertThat(tree.value().is(Tree.Kind.REGULAR_STRING_LITERAL)).isTrue();
+    assertThat(((LiteralTree) tree.value()).value()).isEqualTo("'a'");
+  }
```

Comment:
This test class is about checking how we represent a bit of PHP code.
I wouldn't change this test class as part of this PR.
I believe that [EnumCaseTest](https://github.com/SonarSource/sonar-php/blob/master/php-frontend/src/test/java/org/sonar/php/parser/statement/EnumCaseTest.java) would be a better choice: its goal is to check whether we can parse several bits of PHP code without checking how we represent them.